src/mgr/PyModuleRegistry.h

```diff
@@ -122,11 +122,17 @@ class PyModuleRegistry
     return modules.at(module_name);
   }
 
-  bool module_exists(const std::string &module_name) const
+  /**
+   * Find the specified module.
+   * Returns a null reference if it does not exist.
+   */
+  PyModuleRef find_module(const std::string &module_name)
```

Comment:
It looks right now we have only one user for `get_module` -- `DaemonServer`. So  I would not add a similar function, but modify the existent `get_module`. And then in `DaemonServer`, after the call:
```
PyModuleRef module = py_modules.get_module(handler_name);
```
you could add:
```
ceph_assert(module);
```
So for a reader it would be clear that the function may return empty ref, but the caller does not expect it in this context.